howdoi/howdoi.py

```diff
@@ -97,8 +97,22 @@ def _get_result(url):
         raise e
 
 
+def _add_links_to_text(element):
+    hyperlinks = element.find('a')
+
+    for hyperlink in hyperlinks:
+        pquery_object = pq(hyperlink)
+        href = hyperlink.attrib['href']
+        copy = pquery_object.text()
+        if (copy == href):
+            replacement = copy
+        else:
+            replacement = "[{0}]({1})".format(copy, href)
+        pquery_object.replace_with(replacement)
+
 def _get_text(element):
```

Comment:
I think the *_get_text* can rename to *get_text*..
Because unit test doesn't mean to test for private function.  And a function which is begin with a single underline is considered as a private method or function.

Reply:
Yes let's call it `get_text`.